SAMPLE_FORMAT.md

```diff
@@ -199,6 +202,16 @@ Snippets should have tests that should verify the snippet works and compiles cor
 The following are some general Java best practices that should be followed in samples to remain
 idiomatic. 
 
+### Modern Java
+Prefer using modern ideoms / language features over older styles.
+
+* Lambda's should be about 1-3 lines at most, otherwise it should be in a named method.
+  * Prefer to annonymous classes
+* Streams can be extremely compact, efficient, and easy to use - consider using them.
+  * Avoid side effects
```

Comment:
nit: can you expand on this?

Reply:
I was trying to make it as short as possible, but from the internal java guidelines (see meeting doc).
That said, are you asking me to explain "side effects"?